connect/clients/nats.py

```diff
@@ -44,7 +45,8 @@ async def create_nats_subscribers():
     await start_retransmit_subscriber()
 
     retransmit_loop = asyncio.get_event_loop()
-    retransmit_loop.create_task(retransmitter())
+    global retransmit_task
+    retransmit_task = retransmit_loop.create_task(retransmitter())
```

Comment:
This makes sense. So we will create a task for our "long lived clients" so that we can have a handle to the task. This allows us to send a "cancel" message to the task when we are shutting down.

Reply:
Yes, we were originally ok without performing the cancel, but this is probably a better practice.